src/components/measureGroups/MeasureGroups.tsx

```diff
@@ -44,12 +42,11 @@ import {
 import GroupPopulation from "./GroupPopulation";
 import MeasureGroupScoringUnit from "./MeasureGroupScoringUnit";
 import MeasureGroupObservation from "./MeasureGroupObservation";
+import AutoComplete from "./AutoComplete";
 
 const Grid = styled.div(() => [tw`grid grid-cols-4 ml-1 gap-y-4`]);
-const Content = styled.div(() => [tw`col-span-3`]);
+const Content = styled.div(() => [tw`col-span-3 pl-4 pr-4`]);
 const Header = styled.section`
-  background-color: #f2f5f7;
-  padding: 1em 3em;
   border-bottom: solid 1px rgba(80, 93, 104, 0.2);
 `;
 const Title = styled.h1`
```

Comment:
Looks like we need a h3 and default size and also the color should be #242424. Also the header section supposed to have help text 
* Indicates required field

Reply:
@RohitKandimalla Thanks Rohit. I just pushed in the change for this.